metron-platform/metron-pcap-backend/pom.xml

```diff
@@ -36,10 +36,22 @@
             <version>${guava_version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.zookeeper</groupId>
+            <artifactId>zookeeper</artifactId>
+            <version>${global_zookeeper_version}</version>
+        </dependency>
         <dependency>
             <groupId>org.apache.curator</groupId>
             <artifactId>curator-client</artifactId>
-            <version>2.10.0</version>
```

Comment:
For some reason, we were pulling in curator-client 2.10.0 instead of 2.7.1.  I don't see this causing a problem, but I want to call it out for reviewers.